var/spack/repos/builtin/packages/geant4/package.py

```diff
@@ -86,8 +110,13 @@ def cmake_args(self):
             options.append('-DGEANT4_USE_QT=ON')
             options.append(
                 '-DQT_QMAKE_EXECUTABLE=%s' %
-                spec['qt'].prefix + '/bin/qmake'
-            )
+                spec['qt'].prefix + '/bin/qmake')
+
+        if '+vecgeom' in spec:
+            options.append('-DGEANT4_USE_USOLIDS=ON')
+            options.append('-DUSolids_DIR=%s' % join_path(
+                spec['vecgeom'].prefix,
+                'lib/CMake/USolids'))
```

Comment:
You can actually use `spec['vecgeom'].prefix.lib.CMake.USolids` here instead of `join_path(...)`